bin/express

```diff
@@ -65,7 +66,7 @@ var www = fs.readFileSync(__dirname + '/../templates/js/www', 'utf-8');
     if (empty || program.force) {
       createApplicationAt(path);
     } else {
-      program.confirm('destination is not empty, continue? ', function(ok){
+      prompt.confirm('destination is not empty, continue? ')(function(err,ok){
```

Comment:
If there is an `err`, shouldn't it be handled somehow? `if (!err && ok)` maybe? Not sure what `co-prompt` api is.